client/app/kubernetes/service/kubernetes-service.routes.js

```diff
@@ -28,5 +28,18 @@ angular.module('managerApp')
           value: ['..'],
           format: 'json',
         },
+      })
+      .state('paas.kube.service.update', {
+        url: '/update',
+        views: {
+          kubernetesUpdateView: {
```

Comment:
Can you use a `Component` ? :)

Reply:
The oui-modal is a component by itself. Do you think we need a component here? It just makes use of the modal :)

Reply:
It would make it easier to export your code elsewhere :)

The whole point of `Angular 1.5+` is to have components everywhere we can have them